### PR TITLE
Marks activateService and activateServicePlan public APIs as deprecated.

### PR DESCRIPTION
**Pull request by Mwongela**
Marks the `user/activateServicePlan` and `organization/activateServicePlan` APIs as deprecated. 

Schema update PR: 5583184

The two APIs above will be deprecated from the public beta and then exposed as privileged APIs.

### CODE REVIEW

**Review by eketo-msft — APPROVED** [review]

File: changelog/Microsoft.DirectoryServices.json [hunk 1]
```diff
@@ -9903,6 +9903,32 @@
...
       "SubArea": "Identity and sign-in"
+    },
+    {
+      "ChangeList": [
+        {
+          "Id": "fb19b0a4-779e-11ec-90d6-0242ac120003",
+          "ApiChange": "Method",
+          "ChangedApiName": "activateServicePlan",
+          "ChangeType": "Change",
+          "Description": "Deprecated the [activateServicePlan](https://docs.microsoft.com/en-us/graph/api/user-activateServicePlan?view=graph-rest-beta) method from the [user](https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:9914]: ChangeType fix
```suggestion
          "ChangeType": "Deprecation",
          "Description": "Deprecated the [activateServicePlan](https://docs.microsoft.com/en-us/graph/api/user-activateServicePlan?view=graph-rest-beta) method from the [user](https://docs.microsoft.com/en-us/graph/api/resources/user?view=graph-rest-beta) resource. This method will stop returning data after March 15, 2022.",
```

File: changelog/Microsoft.DirectoryServices.json [hunk 2]
```diff
@@ -9903,6 +9903,32 @@
...
+          "Target": "user"
+        },
+        {
+          "Id": "fb19b0a4-779e-11ec-90d6-0242ac120003",
+          "ApiChange": "Method",
+          "ChangedApiName": "activateService",
+          "ChangeType": "Change",
+          "Description": "Deprecated the [activateService](https://docs.microsoft.com/en-us/graph/api/organization-activateService?view=graph-rest-beta) method to the [user](https://docs.microsoft.com/en-us/graph/api/resources/organization?view=graph-rest-beta) resource.",
```
Comment (FaithOmbongi) [inline changelog/Microsoft.DirectoryServices.json:9922]:
```suggestion
          "ChangeType": "Deprecation",
          "Description": "Deprecated the [activateService](https://docs.microsoft.com/en-us/graph/api/organization-activateService?view=graph-rest-beta) method from the [user](https://docs.microsoft.com/en-us/graph/api/resources/organization?view=graph-rest-beta) resource. This method will stop returning data after March 15, 2022.",
```

File: api-reference/beta/resources/user.md [hunk 3]
```diff
@@ -61,7 +61,7 @@ This resource supports:
 | [List contacts](../api/user-list-contacts.md) | [contact](contact.md) collection | Get a contact collection from the default contacts folder of the signed-in user. |
 | [List contactFolders](../api/user-list-contactfolders.md) | [contactFolder](contactfolder.md) collection | Get the contact folder collection in the default contacts folder of the signed-in user. |
 | **Directory objects**|||
-| [activateServicePlan](../api/user-activateserviceplan.md) | None | Activate a service with a given `servicePlanId` and `skuId` for a given [user](user.md). |
+| [activateServicePlan](../api/user-activateserviceplan.md) (deprecated) | None | Activate a service with a given `servicePlanId` and `skuId` for a given [user](user.md). |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/user.md:64]: Please move it to be the last row in this **Directory objects** section. Approx Line 85. For more info, please see the docs deprecation guidance.

File: api-reference/beta/api/user-activateserviceplan.md [hunk 4]
```diff
@@ -7,12 +7,15 @@ ms.prod: "users"
...
 
-# user: activateServicePlan
+# user: activateServicePlan (deprecated)
 
 Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
+> [!CAUTION]
+> The user `activateServicePlan` API will be deprecated and will stop return data on March 15, 2022.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/user-activateserviceplan.md:17]:
```suggestion
> The **activateServicePlan** API is deprecated and will stop returning data on March 15, 2022.
```

File: api-reference/beta/api/organization-activateservice.md [hunk 5]
```diff
@@ -44,7 +47,7 @@ POST /organization/{organizationId}/activateService
 |Content-Type|application/json. Required.|
 
 ## Request body
-In the request body, supply a JSON representation of the [activateService](../resources/activateService.md) object.
+In the request body, supply a JSON representation of the [activateService](../resources/activateService.md) (deprecated) object.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/organization-activateservice.md:50]: No need to indicate this here. The titles (API + resource) are sufficient.
```suggestion
In the request body, supply a JSON representation of the [activateService](../resources/activateService.md) object.
```

File: api-reference/beta/api/organization-activateservice.md [hunk 6]
```diff
@@ -7,12 +7,15 @@ ms.prod: "directory-management"
...
 
-# organization: activateService
+# organization: activateService (deprecated)
 
 Namespace: microsoft.graph
 
 [!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
 
+> [!CAUTION]
+> The organization `activateService` API will be deprecated and will stop returning data on March 15, 2022.
```
Comment (FaithOmbongi) [inline api-reference/beta/api/organization-activateservice.md:17]:
```suggestion
> The **activateService** API is deprecated and will stop returning data on March 15, 2022.
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
Thank you @Mwongela for working on this PR. Just some minor fixes and please more the changelog to be the first object in the file (starting at Line 3).

File: changelog/Microsoft.DirectoryServices.json (hunk 1 above)
Comment (Mwongela) [reply changelog/Microsoft.DirectoryServices.json]: I have implemented this suggestion.

File: changelog/Microsoft.DirectoryServices.json (hunk 2 above)
Comment (Mwongela) [reply changelog/Microsoft.DirectoryServices.json]: I have implemented this suggestion.

**Comment by FaithOmbongi** [comment]
Closing and reopening to rebuild.

**Review by FaithOmbongi — APPROVED** [review]
LGTM.

**Pull request merged**